deepcave/runs/converters/smac_v2.py

```diff
@@ -0,0 +1,147 @@
+import json
+from pathlib import Path
+
+import numpy as np
+
+from deepcave.runs import Status
+from deepcave.runs.objective import Objective
+from deepcave.runs.run import Run
+from deepcave.utils.hash import file_to_hash
+
+
+class SMAC2Run(Run):
+    prefix = "SMAC2"
```

Comment:
prefix inconsistent with prefix for SMACv1
I would maybe change the prefix to SMAC3v2

Reply:
SMAC2 shouldn't be mistaken with the SMAC2 (the old java version of SMAC) but maybe no one sees this prefix